connectors/email/src/test/java/io/camunda/connector/email/client/jakarta/TestMessage.java

```diff
@@ -13,10 +13,7 @@
 import java.io.InputStream;
 import java.io.OutputStream;
 import java.time.OffsetDateTime;
-import java.util.Date;
-import java.util.Enumeration;
-import java.util.List;
-import java.util.Random;
+import java.util.*;
```

Comment:
Intentional change?

Reply:
I went ahead and removed the wildcard here.

It's an easy performance improvement and make more clear which classes this file is using.